applications/GeoMechanicsApplication/custom_processes/apply_excavation_process.hpp

```diff
@@ -75,7 +73,7 @@ class ApplyExcavationProcess : public Process
     {
         KRATOS_TRY
 
-        const int nElements = mrModelPart.GetMesh(0).Elements().size();
+        const unsigned int nElements = mrModelPart.GetMesh(0).Elements().size();
```

Comment:
```suggestion
        const std::size_t nElements = mrModelPart.NumberOfElements();
```

sizes  => `std::size_t`